src/Tools/dotnet-monitor/Profiler/ProfilerService.cs

```diff
@@ -80,6 +84,22 @@ public async Task ApplyProfiler(IEndpointInfo endpointInfo, CancellationToken ca
                         throw new FileNotFoundException(Strings.ErrorMessage_UnableToFindProfilerAssembly, profilerFileInfo.Name);
                     }
 
+                    // This optional setting instructs where the profiler should establish its socket file
+                    // and where to provide any additional files to dotnet-monitor.
+                    // CONSIDER: Include the runtime instance identifier in the path in order to keep
```

Comment:
Given that the uds files will have the runtime instance id and that we probably want to share the same profiler bits between multiple apps, I'm not sure we want the share folder to have the runtime instance.

Reply:
With the current changes, if DefaultSharedPath is set to `/diag`, then the structure looks like:

```
/diag/dotnet-monitor.sock
/diag/dumps/*
/diag/libs/<ver>/<rid>/native/*MonitorProfiler*
/diag/<rii>.sock
```

If I were to make this change in the future, I think the files that the profiler produces should go into a runtime instance ID folder. The shared library bits would not go into this folder. If the DefaultSharedPath is set to `/diag`, then it would look like this:

```
/diag/dotnet-monitor.sock
/diag/dumps/*
/diag/libs/<ver>/<rid>/native/*MonitorProfiler*
/diag/<rii>/profiler.sock
/diag/<rii>/symbols/*
```